src/algorithms/crypto/randomness.go

```diff
@@ -18,7 +18,7 @@ type Serialization = util.Serialization
 // object, define a struct at the .id level that contains those objects as member fields.
 // This will then cause a Serialize_*() method to be generated for the struct type.
 func (tag DomainSeparationTag) DeriveRand(s Serialization) Randomness {
-	return _deriveRandInternal(tag, s, 0)
+	return _deriveRandInternal(tag, s, -1)
```

Comment:
What's the rationale for changing the sentinel index to -1 instead of 0? I don't have a strong preference, but if it's just aesthetic, would prefer to leave to a subsequent PR for simplicity.

Reply:
0 is genesis block. Ambiguous. -1 is a clear flag

Reply:
It's not at all ambiguous in the precise sense of https://github.com/filecoin-project/specs/pull/679#discussion_r352290621 -- but if it's conceptually confusing, I'm fine either way.

Reply:
nit: it is ambiguous in that it has meaning beyond being a sentinel